app/services/state_file/state_information_service.rb

```diff
@@ -0,0 +1,43 @@
+module StateFile
+  class StateInformationService
+    class << self
+      def active_states # TODO thinking this could also be called `active_state_codes` or just `state_codes`?
```

Comment:
I like `active_state_codes`

Reply:
I agree - the problem is that `state` is such an overloaded term I think we need to be specific. (Is a state the current situation within the program [e.g. Statesman], or a geographical region)